fblldbbase.py

```diff
@@ -73,3 +74,63 @@ def evaluateExpression(expression, printErrors=True):
 
 def evaluateObjectExpression(expression, printErrors=True):
   return evaluateExpression('(id)(' + expression + ')', printErrors)
+
+def evaluateCStringExpression(expression, printErrors=True):
+  ret = evaluateExpression(expression, printErrors)
+
+  process = lldb.debugger.GetSelectedTarget().GetProcess()
+  error = lldb.SBError()
+  ret = process.ReadCStringFromMemory(int(ret, 16), 256, error)
+  if error.Success():
+    return ret
+  else:
+    if printErrors:
+      print error
+    return None
+
+
+RETURN_MACRO = """
+#define IS_JSON_OBJ(obj)\
+    (obj != nil && ((bool)[NSJSONSerialization isValidJSONObject:obj] ||\
+    (bool)[obj isKindOfClass:[NSString class]] ||\
+    (bool)[obj isKindOfClass:[NSNumber class]]))
+#define RETURN(ret) ({\
+    if (!IS_JSON_OBJ(ret)) {\
+        (void)[NSException raise:@"Invalied RETRUN argument" format:@""];\
+    }\
+    NSDictionary *__dict = @{@"return":ret};\
+    NSData *__data = (id)[NSJSONSerialization dataWithJSONObject:__dict options:0 error:NULL];\
+    NSString *__str = (id)[[NSString alloc] initWithData:__data encoding:4];\
+    (char *)[__str UTF8String];})
+#define RETURNCString(ret)\
+    ({NSString *___cstring_ret = [NSString stringWithUTF8String:ret];\
+    RETURN(___cstring_ret);})
+"""
+
+def check_expr(expr):
+  return expr.strip().split('\n')[-1].find('RETURN') != -1
```

Comment:
In case of objective-C code its's better to split by `;` instead of newlines
Also it's probably will be better to check expression by using regexp on the last expression
Something like
```
/\s*RETURN\w*\s*\(.*)\s*\)\s*;\s*/
```
But searching for `RETURN` itself - is, probably fine